include/dbclean.php

```diff
@@ -48,11 +57,18 @@ function remove_orphans($stage = 0) {
 
 	$count = 0;
 
+	// With activated worker we split the deletion in many small tasks
+	if (Config::get("system", "worker")) {
+		$limit = 1000;
+	} else {
```

Comment:
I find this way more readable. I only use the way you suggested when inside the "if" statement there are several "if" statements which would mean that I had to create multiple "else" statements.